[WIP] P5-212: Continous Integration (#131)
aau-hcl-p5/main: report/5.Solution/DockerBuild.tex

In addition to local builds, the Docker image is also used on the TeamCity server, as part of the continuous integration for the project, meaning that the Docker image is used every time something is pushed to a branch.
%Maybe describe Continuous integration?

\subsection{Continous Integration}
%- Intro text

> In the matter ensuring that all of our builds are not failing, the group setup a TeamCity server\cite{TeamCityHomepage}, and combining it with GitHub Webhooks.

Reviewer: ```suggestion
In the matter ensuring that all of our builds are not failing, the group setup a TeamCity server\cite{TeamCityHomepage}, and combining it with GitHub webhooks.
```

> In the matter ensuring that all of our builds are not failing, the group setup a TeamCity server\cite{TeamCityHomepage}, and combining it with GitHub Webhooks.

Reviewer: set up når det er noget aktivt vi gør
```suggestion
To ensure that builds are not failing, the group set up a TeamCity server\cite{TeamCityHomepage}, and combining it with GitHub Webhooks.
```

> A GitHub Webhook is sent from GitHub to the TeamCity server, every time new code has been pushed.

Reviewer: ```suggestion
A GitHub webhook is sent from GitHub to the TeamCity server, whenever new code has been pushed.
```
